cluster/saltbase/salt/kube-addons/initd

```diff
@@ -44,8 +45,31 @@ do_start()
 #
 do_stop()
 {
-    kill $(cat ${PIDFILE})
-    rm ${PIDFILE}
+    # start-stop-daemon is not used because we have to stop all children
+    # limitations:
+    #  - stop does not work if the pid file is missing
+    #  - stop does not work if the daemon process is missing (children will not
+    #    be killed)
+
```

Comment:
Can you add this note into the shell script comments so that it doesn't get lost in github comments.